crates/anthropic/src/anthropic.rs

```diff
@@ -296,10 +296,7 @@ pub async fn stream_completion_with_rate_limit_info(
         .body(AsyncBody::from(serialized_request))
         .context("failed to construct request body")?;
 
-    let mut response = client
-        .send(request)
-        .await
-        .context("failed to send request to Anthropic")?;
```

Comment:
The `context` is helpful for backtraces, so I’m not sure we want to remove it entirely.

Reply:
Whats the best way to keep the context but bubble up the error? Basically Ive seen instances where the API times out and all a user gets to see is 'streaming completion failed'